packages/calling/calling/src/CallingClient/CallingClient.ts

```diff
@@ -225,191 +100,6 @@ export class CallingClient extends Eventing<CallingClientEventTypes> implements
     });
   }
 
-  /**
-   * Fetches countryCode and region of the client.
-   */
-  private async getClientRegionInfo(): Promise<RegionInfo> {
-    const regionInfo = {} as RegionInfo;
-
-    try {
-      const temp = <WebexRequestPayload>await this.webex.request({
-        uri: `${this.webex.internal.services._serviceUrls.mobius}${URL_ENDPOINT}${IP_ENDPOINT}`,
-        method: HTTP_METHODS.GET,
-        headers: {
-          [CISCO_DEVICE_URL]: this.webex.internal.device.url,
-          [SPARK_USER_AGENT]: CALLING_USER_AGENT,
-        },
-        service: ALLOWED_SERVICES.MOBIUS,
-      });
-
-      const myIP = (temp.body as IpInfo).ipv4;
-      const response = <WebexRequestPayload>await this.webex.request({
-        uri: `${DISCOVERY_URL}/${myIP}`,
-        method: HTTP_METHODS.GET,
-        addAuthHeader: false,
-        headers: {
-          [SPARK_USER_AGENT]: null,
-        },
-      });
-
-      const clientRegionInfo = response.body as ClientRegionInfo;
-
-      regionInfo.clientRegion = clientRegionInfo?.clientRegion ? clientRegionInfo.clientRegion : '';
-
-      regionInfo.countryCode = clientRegionInfo?.countryCode ? clientRegionInfo.countryCode : '';
-    } catch (err: unknown) {
-      handleRegistrationErrors(
-        err as WebexRequestPayload,
-        (clientError) => {
-          this.registration.sendMetric(
-            METRIC_EVENT.REGISTRATION_ERROR,
-            REG_ACTION.REGISTER,
-            METRIC_TYPE.BEHAVIORAL,
-            clientError
-          );
-          this.emit(EVENT_KEYS.ERROR, clientError);
-        },
-        {method: GET_MOBIUS_SERVERS_UTIL, file: CALLING_CLIENT_FILE}
-      );
-      regionInfo.clientRegion = '';
-      regionInfo.countryCode = '';
-    }
-
-    return regionInfo;
-  }
-
-  /**
-   * Local method for finding the mobius servers.
-   */
-  private async getMobiusServers() {
```

Comment:
same comment for this

Reply:
This method is used only inside `register()` method which is now present in `Line` class. Some of the common variables are used in both the methods like `primaryMobiusUris` and `backupMobiusUris`. So we have to keep both these in the same file and as `register()` is part of `Line` class so that's why I have moved this method(`getMobiusServers()`) and `getClientRegionInfo()`(which is called from `getMobiusServers()`) to Line.

Reply:
@meta-dipanshu-sharma can we move these functions out of the register() method and instead keep it in CallingClient itself? Then pass the primary and secondary URIs to the Line while initializing it?

Reply:
Done but please check above comment as well.